### PR TITLE
added accessor function to get full path of the associated rekall profile

### PR DESCRIPTION
**Pull request by azizulhakim**

### CODE REVIEW

**Comment by tklengyel** [comment]
This is ok to test.

File: libvmi/accessors.c [hunk 1]
```diff
@@ -211,6 +211,17 @@ vmi_get_name(
     }
 }
 
+char *
+vmi_get_rekall_path(
+    vmi_instance_t vmi){
+    if (VMI_OS_LINUX == vmi_get_ostype(vmi)){
+        return (char*)((linux_instance_t)vmi->os_data)->rekall_profile;
+    }
+    else{
```
Comment (smaresca) [inline libvmi/accessors.c]: Prefer to see a switch/case on ostype, with a default that fails cleanly (explicit return of NULL)

File: libvmi/accessors.c [hunk 2]
```diff
@@ -211,6 +211,17 @@ vmi_get_name(
     }
 }
 
+char *
```
Comment (smaresca) [inline libvmi/accessors.c]: this return type should be a const (also the casts below)

File: libvmi/libvmi.h [hunk 3]
```diff
@@ -553,6 +553,15 @@ page_mode_t vmi_init_paging(
 status_t vmi_destroy(
     vmi_instance_t vmi);
 
+/**
+ * Get full path of associated rekall profile
+ *
+ * @param[in] vmi LibVMI instance
+ * @return Full path of the rekall profile
+ */
+char *vmi_get_rekall_path(
```
Comment (smaresca) [inline libvmi/libvmi.h]: return type const char*

**Comment by azizulhakim** [comment]
Is there any feedback on this pull request? I've already committed the proposed change.

**Comment by tklengyel** [comment]
The PR tester is down at the moment because of hardware issues. Once it's back up we can go ahead and merge this.

**Comment by tklengyel** [comment]
Retest this please.

**Comment by tklengyel** [comment]
This is ok to test.

**Comment by tklengyel** [comment]
Retest this please.

**Comment by tklengyel** [comment]
This branch needs to be rebased to the latest master to resolve conflicts and pass the PR gate.

**Comment by bdpayne** [comment]
retest this please

**Comment by dalevy** [comment]
Did this PR fail?

**Comment by tklengyel** [comment]
Retest this please.

**Comment by tklengyel** [comment]
+1

**Pull request merged**